src/bot/plugins.py

```diff
@@ -16,7 +16,7 @@ class MattermostUserRegistrationInfo:
     last_name: str
 
     @classmethod
-    def from_dict(cls, user_data: dict):
+    def from_dict(cls, user_data: dict[str, str]) -> MattermostUserRegistrationInfo:  # noqa: F821
```

Comment:
Импортируй `from typing import Self`
```suggestion
    def from_dict(cls, user_data: dict[str, str]) -> Self:
```